ui/package-lock.json

```diff
@@ -50,26 +50,26 @@
             "resolved": "https://registry.npmjs.org/adal-node/-/adal-node-0.1.28.tgz",
             "integrity": "sha1-RoxLs+u9lrEnBmn0ucuk4AZepIU=",
             "requires": {
-                "@types/node": "^8.0.47",
-                "async": ">=0.6.0",
-                "date-utils": "*",
-                "jws": "3.x.x",
-                "request": ">= 2.52.0",
-                "underscore": ">= 1.3.1",
-                "uuid": "^3.1.0",
-                "xmldom": ">= 0.1.x",
-                "xpath.js": "~1.1.0"
+                "@types/node": "8.10.38",
+                "async": "2.6.0",
```

Comment:
Need to figure out why your npm is so different. I think @nturinski  is the last person to edit this file. Ask him what version he's on

Reply:
v6.5.0

Reply:
I pushed this from another machine, I'll have to update the npm version